src/Explorer/Controls/Tabs/TabComponent.tsx

```diff
@@ -47,26 +80,32 @@ export class TabComponent extends React.Component<TabComponentProps> {
 
       let className = "toggleSwitch";
       let ariaselected;
+      let tabindex;
       if (index === this.props.currentTabIndex) {
         className += " selectedToggle";
         ariaselected = true;
+        tabindex = 0;
       } else {
         className += " unselectedToggle";
         ariaselected = false;
+        tabindex = -1;
       }
 
       return (
         <div className="tab" key={index}>
-          <AccessibleElement
-            as="span"
+          <span
             className={className}
             role="tab"
-            onActivated={() => this.setActiveTab(index)}
+            onClick={() => this.setActiveTab(index)}
+            onKeyDown={(event: React.KeyboardEvent<HTMLSpanElement>) => this.handlekeypress(event)}
+            onFocus={() => this.setState({ activeTabIndex: index })}
             aria-label={`Select tab: ${tab.title}`}
             aria-selected={ariaselected}
+            tabIndex={tabindex}
+            ref={(element) => (this.tabRefs[index] = element)}
```

Comment:
First of all, it shouldn't be called `tabRefs` since this is just a ref to the tab title, not the entire tab or the tab content. It should be renamed to `tabTitleRefs` to avoid confusion.

Also, if I understand your code correctly, the only reason why you would store the tab title refs is just to focus on the tab title when it's selected. There are other ways you can achieve this. E.g. you can set the `autoFocus` property for the selected tab title.

Reply:
I will redo the solution with the suggestions in the comments